### PR TITLE
Fix deprecation warning for cases with empty hash-like args

### PR DESCRIPTION
**Pull request by bestwebua**
Little addition for #136 (Re-allow Hash-like parameters to be passed to steps). It will fix deprecation warning for cases with empty hash-like args.

Screenshot 2021-06-06 at 3 14 04 PM

### CODE REVIEW

File: spec/integration/transaction_spec.rb [hunk 1]
```diff
@@ -532,6 +532,10 @@ def [](key)
       expect(transaction.call(input)).to be_a Dry::Monads::Result::Success
     end
 
+    it "returns a failure without deprication warning" do
```
Comment (solnic) [inline spec/integration/transaction_spec.rb]:
```suggestion
    it "returns a failure without deprecation warning" do
```

**Review by solnic — COMMENTED** [review]
small typo 🙂

**Comment by solnic** [comment]
Thanks! I'll push a patch release with this.

**Comment by bestwebua** [comment]
> Thanks! I'll push a patch release with this.

Thanks, Piotr! 👍

**Pull request merged**